### DISCUSSION TITLE
no image from Frigate

### DISCUSSION THREAD

**Discussion by pmandrillon** [NickM-27/swatch#69] [Q&A]
Despite several hours of trying everything and anything, I can't access to "latest.jpg" image from Frigate.
My configuration is as follows:
 - Home Assistant installed on a VirtualBox VM
 - address to access HA: http://192.168.0.150:8123
 - Frigate add-on port: 5000
 - one camera configured under the name “camera1”
 - snapshot and recording enabled
 - Swatch add-on installed and functional
 - Swatch integration installed but poorly configured (swatch address never compliant and probably for the same reasons)
With the Internet browser, I have several addresses including the one that seems most relevant to me: http://192.168.0.150:5000/api/camera1.latest.jpg

I ended up thinking that it was necessary to go through an installation like "Docker compose" but my level in Linux is not sufficient and I can't achieve anything good.
Could you tell me where this is going wrong?

Thanks in advance.

**Comment by NickM-27** [comment]
the URL should be http://192.168.0.150:5000/api/camera1/latest.jpg

**Reply by pmandrillon** [reply]
Thank you for confirming the address. It allowed me to explore other avenues.
And I finally found where my problem was coming from. In the Frigate add-on configuration, port 5000 is not exposed by default. By activating it, it works perfectly. I've missed it dozens of times.
Thank you again, I will perhaps come back to you for the rest of Swatch.

I have a suggestion: couldn't port 5000 be redirected by default like 8555? Otherwise, it should be specified in the Frigate documentation on the http API page.

**Reply by NickM-27** [reply]
No, this would be a security issue because 5000 is unauthenticated. So by default it is not exposed directly

**Reply by pmandrillon** [reply]
I should have asked you this question earlier because I would have saved 2 weeks on my project. The positive point is that I learned, a little bit, how Docker works
